SwiftUIStarter/SwiftUIStarter/NativeDisplayView.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  NativeDisplayUIView.swift
+//  SwiftUIStarter
+//
+//  Copyright © 2025 CleverTap. All rights reserved.
+//
+
+import SwiftUI
+import CleverTapSDK
+
+struct NativeDisplayUIView: View {
+    @StateObject private var viewModel = NativeDisplayUIViewModel()
+    @State private var displayUnitID = ""
+    @State private var showingAlert = false
+    @State private var alertMessage = ""
+    @State private var alertTitle = ""
+    
+    var body: some View {
+        NavigationView {
+            VStack(spacing: 0) {
+                VStack(spacing: 16) {
+                    HStack {
+                        Text("NATIVE DISPLAY\nNOTE: CLICKING ON BELOW BUTTON WILL RECORD EVENT WITH SAME NAME.")
+                            .font(.caption)
+                            .fontWeight(.medium)
+                            .foregroundColor(.gray)
+                        Spacer()
+                    }
+                    .padding(.horizontal, 16)
+                    .padding(.top, 16)
+                    
+                    // Display type buttons
+                    HStack(spacing: 20) {
+                        Button("ND_Simple") {
```

Comment:
These do not track events with the same name. The names are `ND_Simple` and  `ND_Carousel` but the events tracked are `NativeDisplaySimple` and `NativeDisplayCarousel`.
Rename the buttons.